skimage/transform/radon_transform.py

```diff
@@ -202,22 +207,36 @@ def iradon(radon_image, theta=None, output_size=None,
     # Construct the Fourier filter
     f = fftfreq(projection_size_padded).reshape(-1, 1)   # digital frequency
     omega = 2 * np.pi * f                                # angular frequency
-    fourier_filter = 2 * np.abs(f)                       # ramp filter
+    # enable the possibily of processing the derivative of the projections
+    if derivative:
+        fourier_filter = np.ones_like(f)                    # differential filter
+    else:
+        fourier_filter = 2 * np.abs(f)                       # ramp filter
+    
+    # Extending the standard filter with frequency cutoff
     if filter == "ramp":
         pass
     elif filter == "shepp-logan":
         # Start from first element to avoid divide by zero
-        fourier_filter[1:] = fourier_filter[1:] * np.sin(omega[1:]) / omega[1:]
+        fourier_filter[1:] = fourier_filter[1:] * np.sin(omega[1:]/(2*freqcutoff)) / (omega[1:]/(2*freqcutoff))
     elif filter == "cosine":
-        fourier_filter *= np.cos(omega)
+        fourier_filter *= np.cos(omega/(2*freqcutoff))
     elif filter == "hamming":
-        fourier_filter *= (0.54 + 0.46 * np.cos(omega / 2))
+        fourier_filter *= (0.54 + 0.46 * np.cos(omega / freqcutoff))
     elif filter == "hann":
-        fourier_filter *= (1 + np.cos(omega / 2)) / 2
+        fourier_filter *= (1 + np.cos(omega/freqcutoff)) / 2
     elif filter is None:
         fourier_filter[:] = 1
     else:
         raise ValueError("Unknown filter: %s" % filter)
+    
+    # Effective frequency cutoff
+    fourier_filter[np.where(2*np.abs(f)>freqcutoff)]=0
+    
+    # Change in the filter to adapte it to projection derivatives
+    if derivative:
+        fourier_filter = np.sign(f)*fourier_filter/(1j*np.pi)
```

Comment:
yes, sorry to be nitpicky about this @jcesardasilva but we try to enforce the PEP8 guidelines as possible (see https://www.python.org/dev/peps/pep-0008/) in order to have style consistency throughout the package.